src/Microsoft.CodeAnalysis.Razor.Workspaces/AcceptedCharacters.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.VisualStudio.LanguageServices.Razor
+namespace Microsoft.CodeAnalysis.Razor
```

Comment:
Fair enough. If that's the case I'd like to go with my proposed solution and make one breaking change that needs to be reacted to.